### ISSUE TITLE
Add patrimoine infos

### ISSUE THREAD

**Issue by awenig** [datanfr/datan#846]
Information sur le patrimoine peut être intéressante (et pas mal de recherches). Des informations sur les pages des députés + des statistiques. Voir si des infos de la HATVP sont intéressantes.

- Ajoutez du texte traitant du sujet du patrimoine dans le texte, soit dans le texte soit sous forme pictogramme sur la fiche du
député. Voir un H3 dans un deuxième temps.

**Comment by remikel** [comment]
Sur HATVP, j'ai testé plusieurs députés mais c'est tout le temps en cours de traitement, et ça fait 3 ans car ils ont été déposé en 2022. Je ne pense pas qu'on puisse utiliser ça avant longtemps.

Il y a marqué que les déclarations sont disponibles à la demande en préfecture, mais ça on peut pas le crawler

**Comment by awenig** [comment]
Oui. Les députés ont dû re-remplir des déclarations après l'élection de 2024. A suivre dans les prochains mois si les déclarations sont mises en ligne par la HATVP.
